src-lib/box.cpp

```diff
@@ -300,14 +300,37 @@ float box_iou(const Darknet::Box & a, const Darknet::Box & b)
 	 */
 
 #if 1
-	const float I = box_intersection(a, b);
-	if (I == 0.0f)
-	{
-		return 0.0f;
-	}
-
-	const float U = box_union(a, b, I);
-
+	const float a_half_w = a.w * 0.5f;
```

Comment:
This is faster than the current implementation?  Please add this to a new PR based on the branch `v5_openblas`.  Don't put the code behind a `#if 1`.  If it is faster, let's use it.